docs/UserGuide.md

```diff
@@ -58,105 +102,154 @@ management tasks done faster than traditional GUI apps.
 
 </div>
 
-### Adding an item: `add`
+### 3.1. Viewing help : `help`
 
-Adds an item to the inventory.
+If you are unsure about the commands and want to seek help, this command opens up this User Guide on your browser 
+**if you are connected to the internet**, where you can get detailed information on how to use each feature in 
+StoreMando. If you are not connected to the internet or have no browser installed on the device you are using to access 
+StoreMando, this command would then open up a pop-up help window which would provide you with the link to this 
+User Guide.
+
+Format: `help`
+
+### 3.2. Adding an item : `add`
+
+Have an item to add to the inventory? Then this command is the right one for you!
 
 Format: `add n/ITEM_NAME l/LOCATION q/QUANTITY [e/EXPIRY_DATE] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 An item can have any number of tags (including 0)
-
 </div>
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 Expiry date of an item is optional.
-
 </div>
 
 Examples:
+* `add n/Chocolate Milk l/kitchen refrigerator q/2` adds 2 Chocolate Milk to the kitchen refrigerator. 
+* `add n/Sunshine Bread l/bedroom q/10 e/2020-01-01 t/favourite` adds 10 Sunshine Bread with expiry date, 2020-01-01, 
```

Comment:
Can we make this example more legit, perhaps change to l/kitchen cupboard 1(somewhere more normal) ... e/2021-04-01(a date that is more updated as of our user guide edited date)

Reply:
Fixed!